### PR TITLE
Update ol version to 8.3.2

### PR DESCRIPTION
**Pull request by oterral**
[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_ol/)

Waiting for https://github.com/openlayers/ol3/pull/4003

@fredj or @elemoine could you have a look to https://github.com/openlayers/ol3/pull/4003 please?

### CODE REVIEW

**Comment by oterral** [comment]
Tests fixed ready for review

File: src/components/geolocation/GeolocationDirective.js [hunk 1]
```diff
@@ -43,8 +43,10 @@ goog.require('ga_throttle_service');
...
         var accuracyFeature = new ol.Feature();
         var positionFeature = new ol.Feature(new ol.geom.Point([0, 0]));
-        var featuresOverlay = new ol.FeatureOverlay({
-          features: [accuracyFeature, positionFeature],
+        var featuresOverlay = new ol.layer.Vector({
+          source: new ol.source.Vector({
+            features: [accuracyFeature, positionFeature]
+          }),
           style: gaStyleFactory.getStyleFunction('geolocation')
         });
```
Comment (elemoine) [inline src/components/geolocation/GeolocationDirective.js]: You can use `useSpatialIndex: false` on the source and `updateWhileAnimating: true` and `updateWhileInteracting: true` on the layer for your vector layer to behave as closely to a feature overlay as possible.

Comment (elemoine) [inline src/components/geolocation/GeolocationDirective.js]: See https://github.com/openlayers/ol3/blob/85815a32a073c30dc3b7aa57fb897401e3d58767/src/ol/interaction/selectinteraction.js#L177-L186 for an example.

Comment (oterral) [inline src/components/geolocation/GeolocationDirective.js]: Thx I 'll do that

**Comment by oterral** [comment]
PR updated

**Comment by oterral** [comment]
jenkins test

**Comment by gjn** [comment]
Do we wait for https://github.com/openlayers/ol3/pull/4016 and https://github.com/openlayers/ol3/pull/4003?

**Comment by oterral** [comment]
It depends, when is the next deploy :)?

**Comment by gjn** [comment]
Not sure. But we can plan to have this in for next deploy. But I think we need https://github.com/openlayers/ol3/pull/4003 in any case, no?

**Comment by oterral** [comment]
yes @elemoine has started a better PR than mine to fix #4003

**Comment by elemoine** [comment]
https://github.com/openlayers/ol3/pull/4023 now merged.

**Comment by oterral** [comment]
OL updated

**Comment by oterral** [comment]
ol3cesium updated to latest

**Comment by fredj** [comment]
The closure-library could be updated as well in the [Makefile](https://github.com/geoadmin/mf-geoadmin3/blob/master/Makefile#L503) to match the version [used in openlayers](https://github.com/openlayers/ol3/blob/master/closure-util.json#L2)

**Comment by fredj** [comment]
> The closure-library could be updated as well in the [Makefile](https://github.com/geoadmin/mf-geoadmin3/blob/master/Makefile#L503) to match the version [used in openlayers](https://github.com/openlayers/ol3/blob/master/closure-util.json#L2)

wrong: the version in the Makefile is correct for openlayers v3.8.2

**Pull request merged**